.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,10 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
 {
 	"name": "C# (.NET)",
-	"image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-jammy",
+	"build": {
+		// Path is relative to the devcontainer.json file.
+		"dockerfile": "devcontainer.dockerfile"
```

Comment:
Is this the recommended path? 
Did you test?

Reply:
https://containers.dev/guide/dockerfile

Reply:
I tested here and it is working.
I did not understand the change very well because the "// Path is relative to the devcontainer.json file."

Reply:
Dockerfile is ambiguous, the docker extension is dockerfile, so I think it's better to use a name, you can have multiple docker files together